mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -159,6 +159,12 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	FIELD field_38805 FROG_VARIANT Lnet/minecraft/class_2378;
 	FIELD field_38806 ROOT_PLACER_TYPE_KEY Lnet/minecraft/class_5321;
 	FIELD field_38807 ROOT_PLACER_TYPE Lnet/minecraft/class_2378;
+	FIELD field_39205 CHAT_TYPE_KEY Lnet/minecraft/class_5321;
```

Comment:
These two should be unmapped (done in other PRs, where the consensus seem to be `MESSAGE_TYPE`)

Reply:
Not sure how I feel about the names not matching the actual registry's own ID (which is `chat_type`), but eh...